content/usgs-packages/Application_Part3.md

```diff
@@ -123,65 +62,93 @@ Show Answer
 ``` r
 layout(matrix(1:3, nrow=3))
 plot(precip_site1$DateTime, precip_site1$precip,
-     col="red", pch=20, xlab = "Time", ylab = "Precip accumulation, in")
+     col="red", pch=20, xlab = "Time", ylab = "Precip accumulation, in",
+     main = paste("Site", sites[1]))
 plot(nitrogen_site1$sample_dt, nitrogen_site1$result_va, 
      col="green", pch=20, xlab = "Time", ylab = "Nitrogren concentration, mg/l")
 plot(phosphorus_site1$sample_dt, phosphorus_site1$result_va,
      col="blue", pch=20, xlab = "Time", ylab = "Phosphorus concentration, mg/l")
 ```
 
-<img src='../static/app-part3/time-series-plots-1.png'/ title='TODO'/>
+<img src='../static/app-part1/time-series-plots-1.png'/ title='TODO'/>
```

Comment:
oops, that's because I didn't commit the image file. The "TODO" is our automatic filler for the title. I'll update that as well